components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -42,6 +42,7 @@ abstract class EngineSession(
         fun onTrackerBlockingEnabledChange(enabled: Boolean) = Unit
         fun onTrackerBlocked(tracker: Tracker) = Unit
         fun onTrackerLoaded(tracker: Tracker) = Unit
+        fun onNavigateBack() = Unit
```

Comment:
Can we implement this for `browser-engine-system` too? :)

Reply:
I've implemented this in SystemEngineSession as well.